js/templates/userPage/userPage.html

```diff
@@ -0,0 +1,58 @@
+<nav id="pageTabBar" class="barLg clrP clrBr">
+
+      <div class="flexVCent pageTabs">
+        <!--consider turning tabs into a reusable view-->
+        <div>
+          <div class="userIcon disc clrBr2 clrSh1" style="background-image: url('../imgs/defaultUser.png')"></div>
+        </div>
+        <div>
+          <h1 class="h2 txUnb txUnl"><%= ob.name %></h1>
+          <div class="txt5b">
+            <span>Stars(x)</span>
+            <span>
+              <% if (ob.followsYou) { %>
+                <%= ob.polyT('userPage.followsYou') %>
+              <% } %>
+            </span>
+          </div>
+        </div>
+        <div class="flexExpand">
+          <div class="flexHRight flexVCent gutterH clrT2">
+            <a class="btn tab clrBr js-tab" data-tab="Home">Home</a>
+            <a class="btn tab clrBr js-tab" data-tab="Store">Store</a>
+            <a class="btn tab clrBr js-tab" data-tab="Following">Following</a>
+            <a class="btn tab clrBr js-tab" data-tab="Followers">Followers</a>
+          </div>
+        </div>
+      </div>
+
+</nav>
+<div class="banner" <% if(ob.banner){ %>
+  style="background-image: url('<%= ob.header %>, url('../imgs/defaultBanner.png')"
+  <% } else { %>
+  style="background-image: url('../imgs/defaultBanner.png')"
```

Comment:
Should it really be doing the if on `ob.banner` and then putting in `ob.header`?

Also, isn't `ob.header` going to be a hash, rather than a full url?

In either case, this is something that was a template helper in v1 and I think we should follow suit in v2. IMHO, it's so much cleaner to call the helper with a few args than have to repeat this type of if / then structure in templates (I've already seen this in some other v2 template).